lib/rules/max-file-line-count.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+/**
+ * Get the 'last' node of the tree
+ *
+ * @param {Object} node - The node whose last child we want to return
+ * @returns {Object} The last node
+ */
+var getLastNode = function (node) {
+  var last = node.last();
+
+  return last ? getLastNode(last) : node;
+};
+
+module.exports = {
+  'name': 'max-file-line-count',
+  'defaults': {
+    length: 300
+  },
+  'detect': function (ast, parser) {
+    var result = [];
```

Comment:
This could be simplified as all we need is the `end.line` of the `stylesheet` node, which will always be `ast.end.line`